agent-inject/agent/annotations.go

```diff
@@ -82,6 +82,26 @@ const (
 
 	// AnnotationAgentInjectToken is the annotation key for injecting the
 	// auto-auth token into the secrets volume (e.g. /vault/secrets/token)
+	// AnnotationAgentInjectTemplateLeftDelim is the key annotation that configures Vault
+	// Agent what left delimiter to use for rendering the secrets.  The name
+	// of the template is any unique string after "vault.hashicorp.com/agent-template-left-delim-",
+	// such as "vault.hashicorp.com/agent-template-left-delim-foobar".  This should map
+	// to the same unique value provided in "vault.hashicorp.com/agent-inject-secret-".
+	// If not provided, a default left delimiter is used as defined by https://www.vaultproject.io/docs/agent/template#left_delimiter
+	AnnotationAgentInjectTemplateLeftDelim = "vault.hashicorp.com/agent-template-left-delim"
+	DefaultLeftDelim                       = "{{"
```

Comment:
I'd rather we set these `DefaultLeftDelim` and `DefaultRightDelim` in config.go; they'll be easier to spot there.